zwave-classifier.js

```diff
@@ -113,10 +113,10 @@ const NOTIFICATION_SENSOR = {
     name: 'overheat',
     propertyName: 'overheat',
     propertyDescr: {
-      '@type': 'AlarmProperty',
+      '@type': 'OverheatProperty',
```

Comment:
The should be AlarmProperty or BooleanProperty, since we don't have an OverheatProperty at this time.